### PR TITLE
CC-24 Add Top Bar Header

### PR DESCRIPTION
**Pull request by skrzepij**

### CODE REVIEW

**Comment by skrzepij** [comment]
[CC-24 Dodac header - top bar](https://trello.com/c/7CZro7vi/54-cc-24-dodac-header-top-bar)

File: src/components/MainHeaderComponent/MainHeaderComponent.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+      <div className="main-header__logo">
+        CConverter
+      </div>
+      <Menu
+        className="main-header__menu"
+        theme={'cc-purple'}
+        mode={'horizontal'}>
+        <Menu.Item key="1">
+          <a href="https://github.com/mad-rat/cconverter" title="Github Repo"  target="_blank" rel="noopener noreferrer">
+            <Icon type="github"/> Github
```
Comment (MateuszWasik) [inline src/components/MainHeaderComponent/MainHeaderComponent.js]: Can we change `Github` to `GitHub` ? Here and in `a title`

File: src/styles/vendors/_antd.scss [hunk 2]
```diff
@@ -1 +1,42 @@
 @import '~antd/dist/antd.css';
+@import "../utils/variables";
+@import "../utils/fonts";
+
+
```
Comment (MateuszWasik) [inline src/styles/vendors/_antd.scss]: whitespace :)

File: src/components/MainHeaderComponent/MainHeaderComponent.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import {Icon, Menu} from 'antd'
+import './MainHeaderComponent.scss'
+
+
```
Comment (MateuszWasik) [inline src/components/MainHeaderComponent/MainHeaderComponent.js]: whitespace :)

**Review by MateuszWasik — APPROVED** [review]
Very good job Radomir !

File: src/components/MainHeaderComponent/MainHeaderComponent.js (hunk 1 above)
Comment (skrzepij) [reply src/components/MainHeaderComponent/MainHeaderComponent.js]: Yes, you can change it because it's out of scope. Elo Essa.

Comment (skrzepij) [reply src/components/MainHeaderComponent/MainHeaderComponent.js]: xD

File: src/styles/vendors/_antd.scss (hunk 2 above)
Comment (skrzepij) [reply src/styles/vendors/_antd.scss]: Eagle eye!

File: src/components/MainHeaderComponent/MainHeaderComponent.js (hunk 1 above)
Comment (skrzepij) [reply src/components/MainHeaderComponent/MainHeaderComponent.js]: Just kidding mate! xD 
 Fixed - thanks! 🦄

**Pull request merged**